docs/backends/clickhouse.mdx

```diff
@@ -0,0 +1,172 @@
+- TTL - You may want to adjust the TTL based on your retention policy. If you need traces for auditing purposes, you may extend it to 365 days. For high-throughput, low-latency systems, you might want to reduce it to 90 days or even less.
+- Partitioning - partitioning by day is a good default, but in high throughput systems you may want to partition by hour or even minute. You may also consider partitioning by service_name, or other attributes.
+- Order By - You may want to adjust the order by clause based on your queries, so that common columns are used first in the query, to optimize performance.
+
+## Configuration
+
+### Endpoint
+
+The ClickHouse endpoint is the URL where the ClickHouse server is listening for incoming connections.
+This is a required setting.
+
+- You can use http, tcp or clickhouse protocols
+- You can use insecure or secure connections (`https://` or `tcp://addr1:port?secure=true`)
+- Can specify multiple host with port: `http://addr1:port,addr2:port`
+- Specify ClickHouse options with query parameters: `http://addr1:port?dial_timeout=5s&compress=lz4`
+
+Please note you are using the correct port for your protocol, defaults are:
+
+- http: 8123
+- tcp: 9000
+- https: 9440
+
+If you are not sure, use the `http` / `https` protocol, as they are more versatile and easier to configure.
+
+### Credentials
+
+If your ClickHouse server requires authentication, you can specify the username and password in the configuration.
+These are optional, keep empty if your ClickHouse server does not require authentication.
+
+### Schema
+
+- Create Schema - Set to `Skip` if you manage your own schema, or `Create` to have Odigos create the schema for you. See [Create Schema](#create-schema) for more details.
+- Database Name (Required) - The name of the Clickhouse Database where the telemetry data will be stored. The default is `otel`. The Database will not be created when not exists, so make sure you have created it before.
+- Table Names - Allows you to customize the names of the tables where the telemetry data will be stored. The default is `otel_traces` for traces and `otel_metrics` for metrics.
+
+## Adding a Destination to Odigos
+
+Odigos makes it simple to add and configure destinations, allowing you to select the specific signals [traces/logs/metrics] that you want to send to each destination. There are two primary methods for configuring destinations in Odigos:
+
+1. **Using the UI**  
+   To add a destination via the UI, follow these steps:
+   - Use the Odigos CLI to access the UI: [Odigos UI](https://docs.odigos.io/cli/odigos_ui)
+   ```bash
+   odigos ui
+    ```
+- In the left sidebar, navigate to the `Destination` page.
+
+- Click `Add New Destination`
+
+- Select `ClickHouse` and follow the on-screen instructions.
+
+2. **Using kubernetes manifests**
+
+Save the YAML below to a file (e.g., `destination.yaml`) and apply it using `kubectl`:
+
+```bash
+kubectl apply -f destination.yaml
```

Comment:
when created from the UI, we put it in the Odigos ns, does it matter here if it will be applied in the default one?